detection-rules/attachment_pdf_linking_to_password_protected_file.yml

```
@@ -0,0 +1,33 @@
+name: "Attachment: Adobe branded PDF file linking to a password-protected file from first-time sender"
+description: |
+    Detects pdf files with links to a remotely hosted password-protected file. This is a common technique
+    abused by Phishing actors as well as Malware actors (IcedID, Remcos, Async Rat)
+type: "rule"
+severity: "high"
+source: |
+  type.inbound 
+  and any(
+      attachments, .file_extension == "pdf" 
+          and any(file.explode(.), 
+              any(ml.nlu_classifier(.scan.ocr.raw).intents,
+              .name == "cred_theft" and .confidence == "high")
+          and strings.icontains(.scan.ocr.raw, "password-protected")
+
+  and any(
+      ml.nlu_classifier(.scan.ocr.raw).entities,
+          .name == "org" and .text == "Adobe")))
+
+  and (
+            (
+                sender.email.domain.root_domain in $free_email_providers
+                and sender.email.email not in $sender_emails
+            )
+            or (
+                sender.email.domain.root_domain not in $free_email_providers
+                and sender.email.domain.domain not in $sender_domains
+            )
+  )
```

```suggestion
    type.inbound 

    and any(attachments, .file_extension == "pdf" 
        and any(file.explode(.), 
            any(ml.nlu_classifier(.scan.ocr.raw).intents,
                .name == "cred_theft" and .confidence == "high"
            )
            
            and strings.icontains(.scan.ocr.raw, "password-protected")
            
            and any(ml.nlu_classifier(.scan.ocr.raw).entities, 
                .name == "org" and .text == "Adobe"
            )
        )
    )

    and (
        (
            sender.email.domain.root_domain in $free_email_providers
            and sender.email.email not in $sender_emails
        )
        or (
            sender.email.domain.root_domain not in $free_email_providers
            and sender.email.domain.domain not in $sender_domains
        )
    )
```

indentation / just making it a bit more readable